recipes-debian/python/python-certifi.inc

```diff
@@ -0,0 +1,15 @@
+SUMMARY = "Python package for providing Mozilla's CA Bundle."
+DESCRIPTION = "This installable Python package contains a CA Bundle that you can reference in your \
+Python code. This is useful for verifying HTTP requests, for example.  This is the same CA Bundle \
+which ships with the Requests codebase, and is derived from Mozilla Firefox's canonical set."
+HOMEPAGE = " http://certifi.io/"
+
+LICENSE = "ISC"
```

Comment:
According to the debian's copyright file, python-certifi is licensed under the Mozilla Public License Version 2.0 ([python-certifi_2018.8.24-1_copyright](https://metadata.ftp-master.debian.org/changelogs//main/p/python-certifi/python-certifi_2018.8.24-1_copyright))

Upstream's LICENSE file indicates it MPS 2.0 too ([LICENSE](https://github.com/certifi/python-certifi/blob/2018.08.24/LICENSE))

Reply:
I fixed it.